packages/firebase_ui_storage/example/functions/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import * as functions from "firebase-functions/v2";
+import { getStorage } from "firebase-admin/storage";
+import { encode } from "blurhash";
+import * as sharp from "sharp";
+import { initializeApp } from "firebase-admin/app";
+
+initializeApp();
+
+type ResizeResult = {
+  buffer: Buffer;
+  width: number;
+  height: number;
+};
+
+exports.genBlurHash = functions.storage.onObjectFinalized(async (event) => {
+  const { bucket: fileBucket, name, contentType } = event.data;
+
+  if (!contentType?.startsWith("image/")) {
```

Comment:
Is this CF supposed to be installed by the user or is this provided as a Firebase Extensions?

Reply:
we will probably put the CF on docs. Blurhash is a kind of opt-in progressive enhancement type of feature anyways